recipes/mongodb_org_repo.rb

```diff
@@ -38,7 +38,7 @@
 when 'rhel', 'fedora'
   yum_repository 'mongodb' do
     description 'mongodb RPM Repository'
-    baseurl "http://downloads-distro.mongodb.org/repo/redhat/os/#{node['kernel']['machine']  =~ /x86_64/ ? 'x86_64' : 'i686'}"
+    baseurl "#{node['mongodb']['repo']}/#{node['kernel']['machine']  =~ /x86_64/ ? 'x86_64' : 'i686'}"
```

Comment:
I've reviewed a little bit the project and Joseph is right, there are a lot of strings with interpolated vars … all over the project

LGTM as well , I'm going to merge this.